.github/workflows/staging-pipeline.yml

```
@@ -35,8 +35,119 @@
+          for i in {1..10}
+          do
+            pg_isready -h localhost -p 5443 -U postgres && echo Success && break
+            echo -n .
+            sleep 1
+          done
+
+      - name: Restore DB backup
+        run: PGPASSWORD=postgres psql -h localhost -p 5443 -U postgres -d givethio < /tmp/backups/givethio-staging/db_backup.sql
+
+      - name: Use Node.js
+        uses: actions/setup-node@v1
```

actionlint (1.7.8)

the runner of "actions/setup-node@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)